### PR TITLE
Pioneer 17 Leg 2 updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
This PR includes Leg 2 mooring deploy/recover & 1 glider recovery updates, as well as Leg 1 AUV deploy/recover updates.
Attachments for reference:
Leg 2 QLR tables
Leg 1 AUV table
Mooring components depth Ss
Oh - also for AUV sensors refer to the premission test docs 3403-40001-00025 and -00026 (in Vault - Platform_Records folder)
[3204-01703_Quick_Look_Cruise_Report_Coastal_Pioneer_17_Leg_2_Tables.pdf](https://github.com/WHOIGit/ooicgsn-asset-management/files/7539459/3204-01703_Quick_Look_Cruise_Report_Coastal_Pioneer_17_Leg_2_Tables.pdf)
Leg 1 AUV table

[Copy of CGSN_Mooring_Component_Depths.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/7539478/Copy.of.CGSN_Mooring_Component_Depths.xlsx)

### CODE REVIEW

File: deployment/CP01CNPM_Deploy.csv [hunk 1]
```diff
@@ -35,12 +35,12 @@ AR48,,AR52,,CP01CNPM-WFP01-02-DOFSTK000,4,1,2020-10-29T20:10:00,2021-04-10T20:29
...
+AR61,,,,CP01CNPM-SBS01-00-STCENG000,5,1,2021-11-10T20:32:00,,CGMCP-01CNPM-00005,,CGCON-BSTC01-00013,40.133907,-70.770433,,0,133,,,
+AR61,,,,CP01CNPM-SBS01-01-MOPAK0000,5,1,2021-11-10T20:32:00,,CGMCP-01CNPM-00005,,CGINS-MOPAK0-11360,40.133907,-70.770433,,0,133,,,
+AR61,,,,CP01CNPM-WFP01-00-WFPENG000,5,1,2021-11-10T20:32:00,,CGMCP-01CNPM-00005,CGVEH-CWFPCP-30002,CGCON-CWFPCP-30002,40.133907,-70.770433,,112,133,,,
+AR61,,,,CP01CNPM-WFP01-01-VEL3DK000,5,1,2021-11-10T20:32:00,,CGMCP-01CNPM-00005,CGVEH-CWFPCP-30002,CGINS-VEL3DK-00470,40.133907,-70.770433,,112,133,,,
+AR61,,,,CP01CNPM-WFP01-02-DOFSTK000,5,1,2021-11-10T20:32:00,,CGMCP-01CNPM-00005,CGVEH-CWFPCP-30002,CGINS-DOFSTK-03523,40.133907,-70.770433,,112,133,,,
+AR61,,,,CP01CNPM-WFP01-03-CTDPFK000,5,1,2021-11-10T20:32:00,,CGMCP-01CNPM-00005,CGVEH-CWFPCP-30002,CGINS-CTDPFK-00185,40.133907,-70.770433,,112,133,,,
+AR61,,,,CP01CNPM-WFP01-04-FLORTK000,5,1,2021-11-10T20:32:00,,CGMCP-01CNPM-00005,CGVEH-CWFPCP-30002,CGINS-FLORTK-01538,40.133907,-70.770433,,112,133,,,
+AR61,,,,CP01CNPM-WFP01-05-PARADK000,5,1,2021-11-10T20:32:00,,CGMCP-01CNPM-00005,CGVEH-CWFPCP-30002,CGINS-PARADK-20609,40.133907,-70.770433,,112,133,,,
```
Comment (reedan88) [inline deployment/CP01CNPM_Deploy.csv:46]: Looks good.

File: deployment/CP02PMCO_Deploy.csv [hunk 2]
```diff
@@ -125,21 +125,21 @@ AR44,,AR52,,CP02PMCO-WFP01-02-DOFSTK000,14,1,2020-06-10T13:53:00,2021-04-11T16:1
...
+AR61,,,,CP02PMCO-SBS01-00-STCENG000,16,1,2021-11-10T13:15:00,,CGMCP-02PMCO-00016,,CGCON-BSTC01-00010,40.100998,-70.887272,,0,146,,,
+AR61,,,,CP02PMCO-SBS01-01-MOPAK0000,16,1,2021-11-10T13:15:00,,CGMCP-02PMCO-00016,,CGINS-MOPAK0-11366,40.100998,-70.887272,,0,146,,,
+AR61,,,,CP02PMCO-WFP01-00-WFPENG000,16,1,2021-11-10T13:15:00,,CGMCP-02PMCO-00016,CGVEH-CWFPCP-10305,CGCON-CWFPCP-10305,40.100998,-70.887272,,125,146,,,
+AR61,,,,CP02PMCO-WFP01-01-VEL3DK000,16,1,2021-11-10T13:15:00,,CGMCP-02PMCO-00016,CGVEH-CWFPCP-10305,CGINS-VEL3DK-00034,40.100998,-70.887272,,125,146,,,
+AR61,,,,CP02PMCO-WFP01-02-DOFSTK000,16,1,2021-11-10T13:15:00,,CGMCP-02PMCO-00016,CGVEH-CWFPCP-10305,CGINS-DOFSTK-02628,40.100998,-70.887272,,125,146,,,
+AR61,,,,CP02PMCO-WFP01-03-CTDPFK000,16,1,2021-11-10T13:15:00,,CGMCP-02PMCO-00016,CGVEH-CWFPCP-10305,CGINS-CTDPFK-00128,40.100998,-70.887272,,125,146,,,
+AR61,,,,CP02PMCO-WFP01-04-FLORTK000,16,1,2021-11-10T13:15:00,,CGMCP-02PMCO-00016,CGVEH-CWFPCP-10305,CGINS-FLORTK-01101,40.100998,-70.887272,,125,146,,,
+AR61,,,,CP02PMCO-WFP01-05-PARADK000,16,1,2021-11-10T13:15:00,,CGMCP-02PMCO-00016,CGVEH-CWFPCP-10305,CGINS-PARADK-20466,40.100998,-70.887272,,125,146,,,
```
Comment (reedan88) [inline deployment/CP02PMCO_Deploy.csv:145]: Looks good.

File: deployment/CP05MOAS-GL559_Deploy.csv [hunk 3]
```diff
@@ -11,9 +11,9 @@ TS20210517,,TS20210809,,CP05MOAS-GL559-02-FLORTM000,2,1,2021-05-17T20:34:00,2021
...
\ No newline at end of file
+TS20210903,,AR61,,CP05MOAS-GL559-00-ENG000000,3,1,2021-09-03T22:53:00,2021-11-13T15:15:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGCON-GLDRCP-00559,39.92405,-70.37323,,1000,1000,SS-2,,
+TS20210903,,AR61,,CP05MOAS-GL559-01-ADCPAM000,3,1,2021-09-03T22:53:00,2021-11-13T15:15:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-ADCPAM-43106,39.92405,-70.37323,,1000,1000,SS-2,,
+TS20210903,,AR61,,CP05MOAS-GL559-02-FLORTM000,3,1,2021-09-03T22:53:00,2021-11-13T15:15:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-FLORTM-03921,39.92405,-70.37323,,1000,1000,SS-2,,
+TS20210903,,AR61,,CP05MOAS-GL559-03-CTDGVM000,3,1,2021-09-03T22:53:00,2021-11-13T15:15:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-CTDGVM-09029,39.92405,-70.37323,,1000,1000,SS-2,,
+TS20210903,,AR61,,CP05MOAS-GL559-04-DOSTAM000,3,1,2021-09-03T22:53:00,2021-11-13T15:15:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-DOSTAM-00364,39.92405,-70.37323,,1000,1000,SS-2; top mount,,
+TS20210903,,AR61,,CP05MOAS-GL559-05-PARADM000,3,1,2021-09-03T22:53:00,2021-11-13T15:15:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00559,CGINS-PARADM-50145,39.92405,-70.37323,,1000,1000,SS-2,,
```
Comment (cdobs) [inline deployment/CP05MOAS-GL559_Deploy.csv:19]: Looks good

File: deployment/CP02PMUO_Deploy.csv [hunk 4]
```diff
@@ -143,21 +143,21 @@ AR48,,AR52,,CP02PMUO-WFP01-02-DOFSTK000,16,1,2020-11-05T15:23:00,2021-04-10T16:0
...
+AR61,,,,CP02PMUO-SBS01-00-STCENG000,18,1,2021-11-09T20:23:00,,CGMCP-02PMUO-00018,,CGCON-BSTC01-00005,39.941533,-70.780857,,0,449,,,
+AR61,,,,CP02PMUO-SBS01-01-MOPAK0000,18,1,2021-11-09T20:23:00,,CGMCP-02PMUO-00018,,CGINS-MOPAK0-11358,39.941533,-70.780857,,0,449,,,
+AR61,,,,CP02PMUO-WFP01-00-WFPENG000,18,1,2021-11-09T20:23:00,,CGMCP-02PMUO-00018,CGVEH-CWFPCP-10301,CGCON-CWFPCP-10301,39.941533,-70.780857,,425,449,,,
+AR61,,,,CP02PMUO-WFP01-01-VEL3DK000,18,1,2021-11-09T20:23:00,,CGMCP-02PMUO-00018,CGVEH-CWFPCP-10301,CGINS-VEL3DK-00035,39.941533,-70.780857,,425,449,,,
+AR61,,,,CP02PMUO-WFP01-02-DOFSTK000,18,1,2021-11-09T20:23:00,,CGMCP-02PMUO-00018,CGVEH-CWFPCP-10301,CGINS-DOFSTK-02708,39.941533,-70.780857,,425,449,,,
+AR61,,,,CP02PMUO-WFP01-03-CTDPFK000,18,1,2021-11-09T20:23:00,,CGMCP-02PMUO-00018,CGVEH-CWFPCP-10301,CGINS-CTDPFK-00124,39.941533,-70.780857,,425,449,,,
+AR61,,,,CP02PMUO-WFP01-04-FLORTK000,18,1,2021-11-09T20:23:00,,CGMCP-02PMUO-00018,CGVEH-CWFPCP-10301,CGINS-FLORTK-01117,39.941533,-70.780857,,425,449,,,
+AR61,,,,CP02PMUO-WFP01-05-PARADK000,18,1,2021-11-09T20:23:00,,CGMCP-02PMUO-00018,CGVEH-CWFPCP-10301,CGINS-PARADK-20610,39.941533,-70.780857,,425,449,,,
```
Comment (reedan88) [inline deployment/CP02PMUO_Deploy.csv:163]: Looks good.

File: deployment/CP03ISPM_Deploy.csv [hunk 5]
```diff
@@ -35,12 +35,12 @@ AR48,,AR52,,CP03ISPM-WFP01-02-DOFSTK000,4,1,2020-10-31T22:02:00,2021-04-12T16:31
...
+AR61,,,,CP03ISPM-SBS01-00-STCENG000,5,1,2021-11-11T13:27:00,,CGMCP-03ISPM-00005,,CGCON-BSTC01-00018,40.362005,-70.878502,,0,93,,,
+AR61,,,,CP03ISPM-SBS01-01-MOPAK0000,5,1,2021-11-11T13:27:00,,CGMCP-03ISPM-00005,,CGINS-MOPAK0-58481,40.362005,-70.878502,,0,93,,,
+AR61,,,,CP03ISPM-WFP01-00-WFPENG000,5,1,2021-11-11T13:27:00,,CGMCP-03ISPM-00005,CGVEH-CWFPCP-30001,CGCON-CWFPCP-30001,40.362005,-70.878502,,69,93,,,
+AR61,,,,CP03ISPM-WFP01-01-VEL3DK000,5,1,2021-11-11T13:27:00,,CGMCP-03ISPM-00005,CGVEH-CWFPCP-30001,CGINS-VEL3DK-00468,40.362005,-70.878502,,69,93,,,
+AR61,,,,CP03ISPM-WFP01-02-DOFSTK000,5,1,2021-11-11T13:27:00,,CGMCP-03ISPM-00005,CGVEH-CWFPCP-30001,CGINS-DOFSTK-03506,40.362005,-70.878502,,69,93,,,
+AR61,,,,CP03ISPM-WFP01-03-CTDPFK000,5,1,2021-11-11T13:27:00,,CGMCP-03ISPM-00005,CGVEH-CWFPCP-30001,CGINS-CTDPFK-00184,40.362005,-70.878502,,69,93,,,
+AR61,,,,CP03ISPM-WFP01-04-FLORTK000,5,1,2021-11-11T13:27:00,,CGMCP-03ISPM-00005,CGVEH-CWFPCP-30001,CGINS-FLORTK-01537,40.362005,-70.878502,,69,93,,,
+AR61,,,,CP03ISPM-WFP01-05-PARADK000,5,1,2021-11-11T13:27:00,,CGMCP-03ISPM-00005,CGVEH-CWFPCP-30001,CGINS-PARADK-20608,40.362005,-70.878502,,69,93,,,
```
Comment (reedan88) [inline deployment/CP03ISPM_Deploy.csv:46]: Looks good.

File: deployment/CP03ISSM_Deploy.csv [hunk 6]
```diff
@@ -372,37 +372,37 @@ AR48,,AR52,,CP03ISSM-SBD11-06-METBKA000,12,1,2020-10-31T19:31:00,2021-03-31T19:0
...
+AR52,,AR61,,CP03ISSM-SBD11-00-DCLENG000,13,1,2021-04-02T20:47:00,2021-11-11T19:30:00,CGMCP-03ISSM-00013,,CGCON-BDCL11-50015,40.3671,-70.881567,,0,93,,,
+AR52,,AR61,,CP03ISSM-SBD11-01-MOPAK0000,13,1,2021-04-02T20:47:00,2021-11-11T19:30:00,CGMCP-03ISSM-00013,,CGINS-MOPAK0-43257,40.3671,-70.881567,,0,93,,,
+AR52,,AR61,,CP03ISSM-SBD11-02-HYDGN0000,13,1,2021-04-02T20:47:00,2021-11-11T19:30:00,CGMCP-03ISSM-00013,,CGINS-HYDGN0-42316,40.3671,-70.881567,,0,93,,,
+AR52,,AR61,,CP03ISSM-SBD11-06-METBKA000,13,1,2021-04-02T20:47:00,2021-11-11T19:30:00,CGMCP-03ISSM-00013,,CGINS-METLGR-00008,40.3671,-70.881567,,0,93,,,
+AR52,,AR61,,CP03ISSM-SBD12-00-DCLENG000,13,1,2021-04-02T20:47:00,2021-11-11T19:30:00,CGMCP-03ISSM-00013,,CGCON-BDCL12-50015,40.3671,-70.881567,,0,93,,,
+AR52,,AR61,,CP03ISSM-SBD12-03-HYDGN0000,13,1,2021-04-02T20:47:00,2021-11-11T19:30:00,CGMCP-03ISSM-00013,,CGINS-HYDGN0-41917,40.3671,-70.881567,,0,93,,,
+AR52,,AR61,,CP03ISSM-SBD12-04-PCO2AA000,13,1,2021-04-02T20:47:00,2021-11-11T19:30:00,CGMCP-03ISSM-00013,,CGINS-PCO2AA-34231,40.3671,-70.881567,,0,93,,,
```
Comment (reedan88) [inline deployment/CP03ISSM_Deploy.csv:405]: Looks good.

File: deployment/CP04OSPM_Deploy.csv [hunk 7]
```diff
@@ -109,21 +109,21 @@ AR44,,AR52,,CP04OSPM-WFP01-02-DOFSTK000,13,1,2020-06-08T13:50:00,2021-04-09T16:0
...
+AR61,,,,CP04OSPM-SBS01-00-STCENG000,15,1,2021-11-09T14:59:00,,CGMCP-04OSPM-00015,,CGCON-BSTC01-00004,39.943353,-70.887295,,0,435,,,
+AR61,,,,CP04OSPM-SBS11-02-MOPAK0000,15,1,2021-11-09T14:59:00,,CGMCP-04OSPM-00015,,CGINS-MOPAK0-11357,39.943353,-70.887295,,0,435,,,
+AR61,,,,CP04OSPM-WFP01-00-WFPENG000,15,1,2021-11-09T14:59:00,,CGMCP-04OSPM-00015,CGVEH-CWFPCP-10303,CGCON-CWFPCP-10303,39.943353,-70.887295,,414,435,,,
+AR61,,,,CP04OSPM-WFP01-01-VEL3DK000,15,1,2021-11-09T14:59:00,,CGMCP-04OSPM-00015,CGVEH-CWFPCP-10303,CGINS-VEL3DK-00036,39.943353,-70.887295,,414,435,,,
+AR61,,,,CP04OSPM-WFP01-02-DOFSTK000,15,1,2021-11-09T14:59:00,,CGMCP-04OSPM-00015,CGVEH-CWFPCP-10303,CGINS-DOFSTK-02725,39.943353,-70.887295,,414,435,,,
+AR61,,,,CP04OSPM-WFP01-03-CTDPFK000,15,1,2021-11-09T14:59:00,,CGMCP-04OSPM-00015,CGVEH-CWFPCP-10303,CGINS-CTDPFK-00126,39.943353,-70.887295,,414,435,,,
+AR61,,,,CP04OSPM-WFP01-04-FLORTK000,15,1,2021-11-09T14:59:00,,CGMCP-04OSPM-00015,CGVEH-CWFPCP-10303,CGINS-FLORTK-01118,39.943353,-70.887295,,414,435,,,
+AR61,,,,CP04OSPM-WFP01-05-PARADK000,15,1,2021-11-09T14:59:00,,CGMCP-04OSPM-00015,CGVEH-CWFPCP-10303,CGINS-PARADK-20468,39.943353,-70.887295,,414,435,,,
```
Comment (reedan88) [inline deployment/CP04OSPM_Deploy.csv:129]: Looks good.

File: deployment/CP05MOAS-GL559_Deploy.csv (hunk 3 above)
Comment (reedan88) [inline deployment/CP05MOAS-GL559_Deploy.csv:19]: Looks good.

File: deployment/CP05MOAS-A6264_Deploy.csv [hunk 8]
```diff
@@ -103,4 +103,11 @@ AR52,,AR52,,CP05MOAS-A6264-02-DOSTAN000,15,1,2021-04-13T19:21:00,2021-04-14T22:1
...
+AR61,,AR61,,CP05MOAS-A6264-01-FLORTN000,16,1,2021-11-04T18:34:00,2021-11-05T20:15:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-FLORTN-01646,39.920783,-70.903067,,,,across-shelf survey,,
+AR61,,AR61,,CP05MOAS-A6264-02-DOSTAN000,16,1,2021-11-04T18:34:00,2021-11-05T20:15:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-DOSTAN-02909,39.920783,-70.903067,,,,across-shelf survey,,
+AR61,,AR61,,CP05MOAS-A6264-03-CTDAVN000,16,1,2021-11-04T18:34:00,2021-11-05T20:15:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-CTDAVN-00527,39.920783,-70.903067,,,,across-shelf survey,,
+AR61,,AR61,,CP05MOAS-A6264-04-NUTNRN000,16,1,2021-11-04T18:34:00,2021-11-05T20:15:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-NUTNRN-01121,39.920783,-70.903067,,,,across-shelf survey,,
+AR61,,AR61,,CP05MOAS-A6264-05-ADCPAN000,16,1,2021-11-04T18:34:00,2021-11-05T20:15:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-ADCPAN-22077,39.920783,-70.903067,,,,across-shelf survey,,
+AR61,,AR61,,CP05MOAS-A6264-06-PARADN000,16,1,2021-11-04T18:34:00,2021-11-05T20:15:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06264,CGINS-PARADN-50267,39.920783,-70.903067,,,,across-shelf survey,,
```
Comment (cdobs) [inline deployment/CP05MOAS-A6264_Deploy.csv:113]: looks good

File: deployment/CP05MOAS-A6263_Deploy.csv [hunk 9]
```diff
@@ -82,4 +82,11 @@ AR52,,AR52,,CP05MOAS-A6263-02-DOSTAN000,12,1,2021-04-13T23:28:00,2021-04-15T02:3
...
+AR61,,AR61,,CP05MOAS-A6263-01-FLORTN000,13,1,2021-11-04T22:35:00,2021-11-06T00:35:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-FLORTN-01645,40.2057,-71.09615,,,,along-shelf survey,,
+AR61,,AR61,,CP05MOAS-A6263-02-DOSTAN000,13,1,2021-11-04T22:35:00,2021-11-06T00:35:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-DOSTAN-02901,40.2057,-71.09615,,,,along-shelf survey,,
+AR61,,AR61,,CP05MOAS-A6263-03-CTDAVN000,13,1,2021-11-04T22:35:00,2021-11-06T00:35:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-CTDAVN-00526,40.2057,-71.09615,,,,along-shelf survey,,
+AR61,,AR61,,CP05MOAS-A6263-04-NUTNRN000,13,1,2021-11-04T22:35:00,2021-11-06T00:35:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-NUTNRN-01109,40.2057,-71.09615,,,,along-shelf survey,,
+AR61,,AR61,,CP05MOAS-A6263-05-ADCPAN000,13,1,2021-11-04T22:35:00,2021-11-06T00:35:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-ADCPAN-22076,40.2057,-71.09615,,,,along-shelf survey,,
+AR61,,AR61,,CP05MOAS-A6263-06-PARADN000,13,1,2021-11-04T22:35:00,2021-11-06T00:35:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-PARADN-50266,40.2057,-71.09615,,,,along-shelf survey,,
```
Comment (cdobs) [inline deployment/CP05MOAS-A6263_Deploy.csv:92]: looks good

File: deployment/CP03ISSM_Deploy.csv (hunk 6 above)
Comment (cdobs) [inline deployment/CP03ISSM_Deploy.csv:405]: looks good

File: deployment/CP05MOAS-A6264_Deploy.csv (hunk 8 above)
Comment (reedan88) [inline deployment/CP05MOAS-A6264_Deploy.csv:113]: Looks good.

File: deployment/CP01CNPM_Deploy.csv (hunk 1 above)
Comment (cdobs) [inline deployment/CP01CNPM_Deploy.csv:46]: looks good

File: deployment/CP05MOAS-A6263_Deploy.csv (hunk 9 above)
Comment (reedan88) [inline deployment/CP05MOAS-A6263_Deploy.csv:92]: Looks good.

**Review by reedan88 — APPROVED** [review]
Looks good.

**Review by cdobs — APPROVED** [review]
looks good!

**Comment by rtravis-whoi** [comment]
[Pioneer 17 AM Updates [DT-151]](https://trello.com/c/LYsz2Yeu/151-pioneer-17-am-updates-dt-151)

**Pull request merged**